PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/ClickToMove.rbxmx

```diff
@@ -219,85 +232,38 @@ local function findPlayerHumanoid(player)
 			return resultHumanoid
 		else
 			humanoidCache[player] = nil -- Bust Old Cache
-			for _, child in pairs(character:GetChildren()) do
-				if child:IsA('Humanoid') then
-					humanoidCache[player] = child
-					return child
-				end
+			local humanoid = character:FindFirstChildOfClass("Humanoid")
+			if humanoid then
+				humanoidCache[player] = humanoid
 			end
+			return humanoid
 		end
 	end
 end
 
-local function CFrameInterpolator(c0, c1) -- (CFrame from, CFrame to) -> (float theta, (float fraction -> CFrame between))
-	local fromAxisAngle = CFrame.fromAxisAngle
+local GetThetaBetweenCFrames; do
```

Comment:
this syntax feels weird, I get the do end scoping, but why not

local GetThetaBetweenCFrames = nil
do
...
end